python-sdk/docs/configurations.rst

```diff
@@ -23,15 +23,103 @@ or by updating Airflow's configuration
    [astro_sdk]
    schema = "tmp"
 
-We can also configure the default schema specific to the database type (example: specific to Snowflake, BigQuery, Postgres). If both the default and database-specific schemas are defined, the preference is given to the database-specific value.
+We can modify all databases by one config ``AIRFLOW__ASTRO_SDK__SQL_SCHEMA``. We can also configure the default schema specific to the database type (example: specific to Snowflake, BigQuery, Postgres). If both the default and database-specific schemas are defined, the preference is given to the database-specific value.
 
 .. code:: python
 
-   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "tmp"
+   AIRFLOW__ASTRO_SDK__POSTGRES_DEFAULT_SCHEMA = "postgres_tmp"
+   AIRFLOW__ASTRO_SDK__BIGQUERY_DEFAULT_SCHEMA = "bigquery_tmp"
+   AIRFLOW__ASTRO_SDK__SNOWFLAKE_DEFAULT_SCHEMA = "snowflake_tmp"
+   AIRFLOW__ASTRO_SDK__REDSHIFT_DEFAULT_SCHEMA = "redshift_tmp"
+
+or by updating Airflow's configuration
+
+.. code:: ini
+
+   [astro_sdk]
+   postgres_default_schema = "postgres_tmp"
+   bigquery_default_schema = "bigquery_tmp"
+   snowflake_default_schema = "snowflake_tmp"
+   redshift_default_schema = "redshift_tmp"
+
+Configuring the unsafe dataframe storage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The dataframe are pickled in astro-sdk-python and saved in airflow database. The dataframe is defined by the user and if it is big enough it might potentially break Airflow's metadata. Hence, unsafe dataframe storage should be set to True.
+
```

Comment:
```suggestion
The dataframes are pickled in astro-sdk-python and saved in the airflow metadata database. Since this dataframe is defined by the user and if it is huge, it might potentially break Airflow's metadata DB by using all the available resources. Hence, unsafe dataframe storage should be set to ``True`` once you are aware of this risk and are OK with it.

```